code/__DEFINES/jobs.dm

```diff
@@ -30,8 +32,8 @@
 
 #define DEPARTMENT_GREYSON "Greyson Positronic"
 
-#define ALL_DEPARTMENTS list(DEPARTMENT_COMMAND, DEPARTMENT_MEDICAL, DEPARTMENT_ENGINEERING, DEPARTMENT_SCIENCE, DEPARTMENT_SECURITY, DEPARTMENT_LSS, DEPARTMENT_CIVILIAN, DEPARTMENT_CHURCH, DEPARTMENT_PROSPECTOR, DEPARTMENT_INDEPENDENT)
+#define ALL_DEPARTMENTS list(DEPARTMENT_COMMAND, DEPARTMENT_MEDICAL, DEPARTMENT_ENGINEERING, DEPARTMENT_SCIENCE, DEPARTMENT_SECURITY, DEPARTMENT_BLACKSHIELD, DEPARTMENT_LSS, DEPARTMENT_CIVILIAN, DEPARTMENT_CHURCH, DEPARTMENT_PROSPECTOR, DEPARTMENT_INDEPENDENT)
 
 #define ASTER_DEPARTMENTS list(DEPARTMENT_COMMAND, DEPARTMENT_LSS)
 
-#define JOBS_OVERALL list(JOBS_SECURITY, JOBS_COMMAND, JOBS_ENGINEERING, JOBS_MEDICAL, JOBS_SCIENCE,JOBS_LSS,JOBS_CIVILIAN,JOBS_CHURCH,JOBS_PROSPECTOR)
+#define JOBS_OVERALL list(JOBS_COMMAND, JOBS_ENGINEERING, JOBS_MEDICAL, JOBS_SCIENCEm JOBS_SECURITY, JOBS_BLACKSHIELD, JOBS_LSS, JOBS_CIVILIAN, JOBS_CHURCH,JOBS_PROSPECTOR)
```

Comment:
`JOBS_SCIENCEm`, should be probably `JOBS_SCIENCE,`

Reply:
Also no space in-between `JOBS_CHURCH,JOBS_PROSPECTOR`

Reply:
Typo moment